### DISCUSSION TITLE
Web Interface is Not Working on 2.3.91

### DISCUSSION THREAD

**Discussion by ns4444-8** [Security-Onion-Solutions/securityonion#6880] [Unsupported Versions]
Hello, 

I am new to Security Onion (SO) and getting a hang of it. I have version 2.3.91. The web interface is not starting on my Windows 10 VM and I know it is a networking misconfiguration. I tried several things and am not sure what to do now, other than reinstall from ISO image and choose DHCP configuration. 

I have two NICS—one NAT (for internet access) and the other NATNetwork (for monitor mode). I already have a NATNetwork established that I have 4 VM on the same subnet (10.0.2.x). I allowed the IP addresses of 10.0.2.0/24 through the terminal command of "sudo so-allow". The SO NAT IP address was set to 10.0.2.10 and it is not getting pinged by my Windows VM that is on the NATNetwork. I tried changing SO IP to 10.0.2.11 with "NEW_IP=y.y.y.y so-ip-update" and the new IP address still does not work. Then, I tried changing the SO subnet by choosing an IP address of 10.0.3.11, but to no avail. I am guessing that I chose the wrong network card for the configuration. If so, is there any way to change the network card from the terminal with a command? 

Thank you in advance!!

**Comment by RobertClum** [comment]
What virtualization software are you using (i.e. VMware, Hyper-V, Oraclebox)?

Have you tried bridge settings?

**Reply by ns4444-8** [reply]
Thank you for responding. 

I am using Oraclebox. I have not tried the bridged setting because I have a hacking lab that I would like Security Onion to monitor. Bridging it will not be ideal. However, I can try it. Can I just change the NIC through Security Onion Oraclebox interface setting and it will automatically change in Security Onion? I believe I tried to change the NIC cards before through the described method but to no avail.

**Comment by RobertClum** [comment]
If I am understanding you correctly, you have systems on two separate NATs one for SO and the other for the systems you want to monitor. since those systems are isolated to their own NAT'd networks, you will need to make sure the monitoring NIC is able to talk to that NAT which contains the systems to be monitored.

I think this video might help explain how the configuration should be setup in Oraclebox to get those two separate NATs communicating.

How to Make Virtual Machines Talk to Each Other in VirtualBox:
https://www.youtube.com/watch?v=vReAkOq-59I

**Reply by ns4444-8** [reply]
Hello Robert, 

Thank you for your reply. The problem is—I am not able to ping SO from my home lab network, which I followed online SO youtube tutorials to set up the NICs. However, SO is not able to be pinged on my home lab network, which tells me that SO is not a part of my home lab network. If you read my first post up top, you can gain a better understanding of the issue. I will keep messing around to find a solution later today. Just hoping to get some insight into what I am doing wrong 😅.

**Reply by RobertClum** [reply]
Okay, check out Oracle host only network article as well

[https://docs.oracle.com/en/virtualization/virtualbox/6.0/user/network_hostonly.html](url)
